### PR TITLE
Create Accessibility Popover

### LINKED ISSUE #61: ♿ Accessibility Modal
**Type**: client
**FOF**:  ❓ 

## Description

Improve the accessibility of the application by making the base text sizes and color scheme of the website customizable. This will require the ability to change the color mode by configuring the theme of the application properly at a high level and a component by component basis. Additionally will need a draggable input that acts as a multiplier for the websites base font sizes.

## Screenshot

image

## Todo

- [ ] High Contrast color scheme 
- [ ] Draggable font size
- [ ] Global language setting? -- dependent on if Brink wants to support this in Contentful.

## Links
- [Chakra UI Color Mode](https://chakra-ui.com/docs/styled-system/features/color-mode)
- [Chakra UI Font Sizes](https://chakra-ui.com/docs/styled-system/features/text-and-layer-styles)
- [Chakra UI RTL](https://chakra-ui.com/docs/styled-system/features/rtl-support)
- [Figma](https://www.figma.com/file/6aJ3cfu2Yq5dxUQcNDkm6Y/Brink-Designs)

### PR DESCRIPTION
**Pull request by code-arman**
## Status:

:rocket: Ready

## Summary

Closes #61 

## Changes

- Creates Popover that overlays the Options button
- Deletes all nav bar links except Option for desktop view
- Utilizes aria labels to make modal compatible with screen reader

## Screenshots
Screen Shot 2022-04-26 at 5 33 58 PM

### CODE REVIEW

File: client/src/components/popover/OptionsPopover.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  SliderTrack,
+  Stack,
+  Switch,
+  Text,
+  useBoolean,
+} from '@chakra-ui/react';
+import React, { FC, ReactElement } from 'react';
+
+export const OptionsPopover: FC = ({ children }): ReactElement => {
+  const [flag, setFlag] = useBoolean();
```
Comment (ayanmallik) [inline client/src/components/popover/OptionsPopover.tsx:25]: Let's rename this boolean to specify what it represents - how about `isOnHighContrast`?

File: client/src/components/popover/OptionsPopover.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    <Popover>
+      <PopoverTrigger>{children}</PopoverTrigger>
+      <PopoverContent marginRight={5}>
+        <PopoverArrow />
+        <PopoverCloseButton />
+
+        <PopoverBody padding={8}>
+          <Stack spacing="30px">
+            <Box>
+              {/* TODO: I was thinking we should use exact font sizes instead of textStyle
```
Comment (ayanmallik) [inline client/src/components/popover/OptionsPopover.tsx:36]: I completely agree - everything in the accessibility modal itself should have fixed font sizes. We can probably remove this comment now

File: client/src/components/nav/NavDesktopLinks.tsx [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
+import { Button, Icon } from '@chakra-ui/react';
 import React, { Fragment, ReactElement } from 'react';
 import AccessibilityNewIcon from '@mui/icons-material/AccessibilityNew';
 
-import { NavItem, NavLink } from '../../components';
+import { NavItem } from '../../components';
+import { OptionsPopover } from '../popover/OptionsPopover';
 
 export const NavDesktopLinks = (): ReactElement => (
   <Fragment>
```
Comment (ayanmallik) [inline client/src/components/nav/NavDesktopLinks.tsx:9]: Now that we've consolidated the navbar - you can remove this unnecessary `<Fragment />`.

**Review by ayanmallik — APPROVED** [review]
Left some quick fixes - but overall, this accessibility popover (good find btw!) looks super clean and I love the interaction you enabled to make this component accessibility. After merging this in, we'll want to refactor the tab list to account for the addition of the "Help" tab.

**Review by amit-sawhney — APPROVED** [review]

**Pull request merged**